lib/money/money.rb

```diff
@@ -1,7 +1,9 @@
 class Money
   include Comparable
+  extend Forwardable
 
   attr_reader :value, :subunits, :currency
+  def_delegators :@value, :zero?, :nonzero?, :positive?, :negative?, :to_i, :to_f, :hash
```

Comment:
may want to delegate to `to_d` instead or @value, always feels weird to delegate to an instance varibable

Reply:
Not that weird, it's in [the docs](http://ruby-doc.org/stdlib-2.3.0/libdoc/forwardable/rdoc/Forwardable.html) 😄